### PR TITLE
Added create more button in the asset creation form

### LINKED ISSUE #3881: Add create more button in the asset creation form
**Describe the bug**

when we are adding multiple assets to the facility, each time the asset is created, it is been redirected to facility page and we need to click create asset button each time

**To Reproduce**

https://care.coronasafe.in/facility/be6211ba-9057-41eb-9a0d-b1800d76c2e7/assets/new

**Expected Behavior**

- [ ] Implement create more button along with the create button in the asset creation form similar to the bed capacity button in the facility page 

**Screenshot**

image

image

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #3881 

Added a create and add more assets button
http://localhost:4000/facility/be6211ba-9057-41eb-9a0d-b1800d76c2e7/assets/new
image

https://www.loom.com/share/048a45f065d64b75886794ffbb0120de

### CODE REVIEW

**Comment by nihal467** [comment]
@Pranshu1902 
image

- [ ] when i use the add more asset button for adding second asset, the both date field in the warranty and service are showing invalid date format

**Comment by Pranshu1902** [comment]
@nihal467 
Bug resolved
image

**Comment by nihal467** [comment]
@Pranshu1902 @khavinshankar LGTM, A small enhancement issue for this PR has been filed on 3929,

File: src/Components/Facility/AssetCreate.tsx [hunk 1]
```diff
@@ -279,7 +282,30 @@ const AssetCreate = (props: AssetProps) => {
...
+            setSupportName("");
+            setSupportEmail("");
+            setSupportPhone("");
+            setQrCodeId("");
+            setManufacturer("");
+            setWarrantyAmcEndOfValidity("");
+            setLastServicedOn("");
+            setNotes("");
+            setWarrantyAmcEndOfValidity(null);
+            setLastServicedOn(null);
```
Comment (khavinshankar) [inline src/Components/Facility/AssetCreate.tsx]: can this be moved into a function, something like `resetFilters`

**Review by khavinshankar — CHANGES_REQUESTED** [review]
minor refactor

File: src/Components/Facility/AssetCreate.tsx (hunk 1 above)
Comment (Pranshu1902) [reply src/Components/Facility/AssetCreate.tsx]: @khavinshankar Done

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**